edi_purchase_oca/models/purchase_order.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+
+
+class PurchaseOrder(models.Model):
+    _name = "purchase.order"
+    _inherit = ["purchase.order", "edi.exchange.consumer.mixin"]
+
+    def button_confirm(self):
```

Comment:
this part is obsolete as we  have specific events now or even the possibility to not use components.

Reply:
this part is missing https://github.com/OCA/edi/pull/1098